app/addons/documents/queryoptions/queryoptions.react.jsx

```diff
@@ -411,7 +405,8 @@ var QueryTray = React.createClass({
             reduce={this.props.reduce}
             toggleReduce={Actions.toggleReduce}
             groupLevel={this.props.groupLevel}
-            updateGroupLevel={Actions.updateGroupLevel} />
+            updateGroupLevel={Actions.updateGroupLevel}
+            docURL={FauxtonAPI.constants.DOC_URLS.GENERAL} />
```

Comment:
This should be changed to `docUrl={this.props.docURL}` then you can override this via `FauxtonAPI.constants.DOC_URLS.GENERAL` or via the `props` for this component.